aws-gitlab-main.yml

```diff
@@ -290,6 +313,38 @@ Resources:
         - !Ref GitLabPublicSubnetA
         - !Ref GitLabPublicSubnetB
 
+  #RDS POSTGRESQL DATABASE
+  GitLabDBSubnetGroup: 
+    Type: AWS::RDS::DBSubnetGroup
+    Properties:
+      DBSubnetGroupDescription: Group includes GitLabPrivateSubnetA and GitLabPrivateSubnetB
+      #DBSubnetGroupName: #may not need
+      SubnetIds:
+        - !Ref GitLabPrivateSubnetA
+        - !Ref GitLabPrivateSubnetB 
+
+  GitLabDB:
+    Type: AWS::RDS::DBInstance 
+    Properties: 
+      AllocatedStorage: !Ref GitLabAllocatedStorageParameter #provisioned IOPS (SSD) for storage type?
+      AutoMinorVersionUpgrade: False
+      #BackupRetentionPeriod:     #preferred backup settings == ? 
+      #DBClusterIdentifier: #related to Redis Cluster later?
```

Comment:
No its asking for this: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-rds-dbcluster.html

Ignore if you are not creating DB cluster